Source: written by heirecka
Upstream: yes, upstream Ticket #7808
Reason: gnutls_transport_set_lowat has been removed from >gnutls-3
Index: src/engine/tlssocket.cpp
===================================================================
--- src/engine/tlssocket.cpp	(revision 4300)
+++ src/engine/tlssocket.cpp	(working copy)
@@ -113,7 +113,9 @@
 	gnutls_transport_set_push_function(m_session, PushFunction);
 	gnutls_transport_set_pull_function(m_session, PullFunction);
 	gnutls_transport_set_ptr(m_session, (gnutls_transport_ptr_t)this);
+#if GNUTLS_VERSION_NUMBER < 0x020c00
 	gnutls_transport_set_lowat(m_session, 0);
+#endif
 
 	m_shutdown_requested = false;
 
